recipe/meta.yaml

```diff
@@ -11,46 +11,64 @@ source:
   sha256: 2ad7f0c8010ebbc46319cc30febfb8e59ccf84969a201541c207bc3a4fa6cf81
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
 
-requirements:
-  host:
-    - python >=3.8
-    - pip
-    - hatchling
-  run:
-    - packaging
-    - python >=3.8
-    - importlib-metadata
-    - typing-extensions >=4.0.1
-    - jinja2
-    - jsonschema >=3.0
-    - numpy
-    - pandas >=0.25
-    - toolz
+outputs:
+  - name: altair
+    requirements:
+      host:
+        - python >=3.8
+        - pip
+        - hatchling
+      run:
+        - packaging
+        - python >=3.8
+        - importlib-metadata
+        - typing-extensions >=4.0.1
+        - jinja2
+        - jsonschema >=3.0
+        - numpy
+        - pandas >=0.25
+        - toolz
+    test:
+      source_files:
+        - altair
+      requires:
+        - pytest
+        - sphinx
+        - docutils
+        - vega_datasets >=0.9.0
+        - anywidget >=0.9.0
+        - recommonmark
+        - ipython
+        - pip
+      commands:
+        - pip check
+        - python -m pytest --pyargs --doctest-modules altair
+      imports:
+        - altair
+        - altair.expr
+        - altair.utils
+        - altair.vegalite
 
-test:
-  source_files:
-    - altair
-  requires:
-    - pytest
-    - sphinx
-    - docutils
-    - vega_datasets
-    - recommonmark
-    - ipython
-    - pip
-    - anywidget
-  commands:
-    - pip check
-    - python -m pytest --pyargs --doctest-modules altair
-  imports:
-    - altair
-    - altair.expr
-    - altair.utils
-    - altair.vegalite
+  - name: altair-all
+    requirements:
+      host:
+        - python >=3.8
+      run:
+        - vl-convert-python >=1.1.0
+        - pyarrow >=11
+        - vegafusion-python-embed >=1.5.0
+        - {{ pin_subpackage('altair', exact=True) }}
```

Comment:
I copied this from [the seboarn recipe](https://github.com/conda-forge/seaborn-feedstock/blob/main/recipe/meta.yaml) and saw that matplotlib has something similar although slightly different. Let me know if this looks reasonable

Reply:
nice, I hadn't seen this before